pkg/controller/iotproject/device_registry_tenant_cleaner.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package iotproject
+
+import (
+	"github.com/enmasseproject/enmasse/pkg/controller/iotconfig"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+)
+
+
+func createIotTenantCleanerJob(name string, iotconfig iotconfig) batchv1.Job {
+
+	job := batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{Name: "iot-registry-tenant-cleaner"},
```

Comment:
This would be unique, and so we need another way of naming. Otherwise we could have only one cleaner. However it should also not be random, as that might result in multiple cleanup jobs for one tenant … maybe use the unique tenant + timestamp … in form that kubernetes allows it.